tensorflow/python/ops/variables.py

```diff
@@ -226,7 +226,12 @@ def __init__(self,
           expected_shape=expected_shape)
 
   def __str__(self):
-    return str(self._snapshot)
+    return "<Variable name='%s' shape=%s dtype=%s>" % (
+            self.name(), self.get_shape(), self.dtype().name)
```

Comment:
`Variable.dtype` is a property, not a method though, so I think `self.dtype` is required.

Can you also remove `name=` from the repr for consistency with the other reprs?